packages/client/hmi-client/src/services/project.ts

```diff
@@ -102,7 +102,8 @@ async function getAssets(projectId: string, types?: string[]): Promise<ProjectAs
 			});
 		} else {
 			Object.values(AssetType).forEach((type, indx) => {
-				url += `${indx === 0 ? '?' : '&'}types=${type}`;
+				// FIX: right now the we cannot get a documents assets, this condition is a temporary measure so that this endpoint doesnt break until we can get documents assets
```

Comment:
Do you mind opening an issue in github so we remember to change this once TDS makes the change in their end?

Reply:
Yup! Opened this last week still in progress: https://github.com/DARPA-ASKEM/data-service/issues/333

Reply:
Refer to this issue for this change on our side: https://github.com/DARPA-ASKEM/terarium/issues/1954